src/utils/constants.js

```diff
@@ -40,61 +40,62 @@ export const WEBCLIENT = "WEBCLIENT";
  * @type {Object}
  */
 export const REQUEST_PARAMS = {
-    DATASET_ID : 'DATASET',
-    SERVER : 'SERVER',
-    IMAGE_ID : 'IMAGE_ID',
-    CHANNELS : 'C',
-    PLANE : 'Z',
-    TIME : 'T',
-    PROJECTION : 'P',
-    MODEL : 'M',
-    CENTER_X : 'X',
-    CENTER_Y : 'Y',
-    ZOOM : 'ZM'
+    CHANNELS: 'C',
```

Comment:
Not possible to use the constants defined in ol3-viewer/src/ome/ol3/globals.js
only diff is ``DATASET_ID``?